bot/exts/github/github.py

```diff
@@ -0,0 +1,83 @@
+import typing
+
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from . import _issues, _profile, _source
+
+
+class Github(commands.Cog):
+    """Github."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+
+    @commands.group(name="github", aliases=("gh",), invoke_without_command=True)
+    async def github_group(self, ctx: commands.Context) -> None:
+        """Commands for Github."""
+        await ctx.send_help(ctx.command)
+
+    @github_group.command(name="profile")
+    @commands.cooldown(1, 10, BucketType.user)
+    async def profile(
+        self, ctx: commands.Context, username: typing.Optional[str]
+    ) -> None:
+        """
+        Fetches a user's GitHub information.
+
+        Username is optional and sends the help command if not specified.
+        """
+        github_profile = _profile.GithubInfo(self.bot.http_session)
+        embed = await github_profile.get_github_info(username)
+
+        if embed is None:
+            await ctx.send_help(ctx.command)
+            ctx.command.reset_cooldown(ctx)
+            return
```

Comment:
Let's handle the absence of username here instead of [get_github_info](https://github.com/gurkult/gurkbot/pull/66/files#diff-47f6432ddf370ef36c67c3320cc9bd1fea2e26bf87bc1dd1f8edc9ae21d53ddfR91-R92).
```suggestion
        if username is None:
            raise MissingRequiredArgument
  
        github_profile = _profile.GithubInfo(self.bot.http_session)
        embed = await github_profile.get_github_info(username)           
```

[1/2](https://github.com/gurkult/gurkbot/pull/66/files#diff-47f6432ddf370ef36c67c3320cc9bd1fea2e26bf87bc1dd1f8edc9ae21d53ddfR85-R92)